src/components/MoviesList/MoviesList.tsx

```diff
@@ -3,20 +3,20 @@ import './MoviesList.scss';
 import { MovieCard } from '../MovieCard';
 
 interface Props {
-  movies: Movie[];
+  movies: Movie[] | null;
 }
 
 export const MoviesList: React.FC<Props> = (props) => {
   const { movies } = props;
 
   return (
     <div className="movies">
-      {movies.map(movie => (
+      {movies && (movies.map(movie => (
```

Comment:
movies always an array (it will not be false ever, even if it will be empty). Use another check